### ISSUE TITLE
add assemble and scan command to mdraid howto

### ISSUE THREAD

**Issue by phillxnet** [rockstor/rockstor-doc#177]
Thanks to forum member HBDK in the following forum thread for highlighting this issue. As of Rockstor iso 3.9.1 when following the docs "Mirroring Rockstor OS using Linux Raid" howto the mdraid system no longer automatically assembles the md devices. It was found that an additional command in now required to enact the raid devices scan and assembly:

step 8: Change our root mdraid to btrfs

has
```
cat /proc/mdstat
mkfs.btrfs -f -L rockstor_rockstor /dev/md###
exit
```
this should now read:
```
mdadm --assemble --scan
cat /proc/mdstat
mkfs.btrfs -f -L rockstor_rockstor /dev/md###
exit
```

It would also be nice to have the image that directly follows on from these commands also updated.

Please update the following forum thread with this issues resolution:
https://forum.rockstor.com/t/error-with-mirroring-rockstor-os/4083

**Comment by HBDK** [comment]
i have created a pull request for this and some other changes
#178

**Comment by phillxnet** [comment]
Belatedly closing as fixed by @HBDK in pull request #178.
Thanks again @HBDK for fixing this one.
